src/Compilers/VisualBasic/Portable/Symbols/Source/SourceIncompleteMemberFieldSymbol.vb

```diff
@@ -0,0 +1,44 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    Friend NotInheritable Class SourceIncompleteFieldSymbol
```

Comment:
>SourceIncompleteFieldSymbol [](http://example.com/codeflow?start=32&length=27)

Is removal of this symbol still work in progress?

Reply:
The whole VB part is work in progress. I want to get C# part of the change in a good state first.